lib/presentation/organisms/timer_organism.dart

```diff
@@ -24,6 +26,10 @@ class _TimerOrganismState extends State<TimerOrganism>
   @override
   void initState() {
     super.initState();
+    lockScreen = _prefs.getBool("isLockScreen") ?? lockScreen;
+    WakeLockController.instance.setWakeLock(lockScreen);
+    PlayerController.instance.setSilentState(!(_prefs.getBool("isSound") ?? true));
```

Comment:
isSound is good, let's change setSilentState and isSilent to it as well.
This will be clearer and the code will be simpler

Reply:
Updated.